ui/src/entities/User/selectors.ts

```diff
@@ -0,0 +1,22 @@
+import { useSelector } from 'react-redux';
+
+import { RootState } from 'src/duck';
+import { AuthState } from 'src/duck/auth';
+import { EmployeeState } from 'src/duck/employee';
+import { ProjectState } from 'src/duck/project';
+
+export const getUsersOrganizationId = () => {
+  const { activeEmployeeId } = useSelector<RootState, AuthState>((state) => state.auth);
+  const { employees } = useSelector<RootState, EmployeeState>((state) => state.employees);
+
+  const emp = activeEmployeeId?.length ? employees.find((e) => e.ecode === activeEmployeeId) : null;
+  return emp?.organization_id ? emp.organization_id : null;
+};
+
+export const getUsersProjects = () => {
```

Comment:
`Users` shouldn't be a plural here. This function (I suppose) returns active projects of current user. Projects are many, but user isn't.